views/layout.html

```diff
@@ -45,6 +45,8 @@
   var mailpoet_date_format = "<%= wp_datetime_format()|escape('js') %>";
   var mailpoet_time_format = "<%= wp_time_format()|escape('js') %>";
   var mailpoet_version = "<%= mailpoet_version() %>";
+  var mailpoet_locale = "<%= mailpoet_locale() %>";
+  var mailpoet_settings = <%= json_encode(settings) %>;

```

Comment:
I think it is not good that we have settings dumped like that into our template. I think it is not ok that we have it in newsletters.html template but moving it to global template makes it worse.
1) we use settings more like a common key value storage and it may contain some passwords
2) As you've mentioned in slack. It has inconsistencies in boolean and also othet types.
3) We also store some logs within the settings so it may contain larger amount of data.

I think we could use some global settings object but I would rather create it within PHP with whitelisted properties and correct (fixed) data types. But this is out of scope of this task and requires some discussion.

To fix it for this issue I suggest you to set into the template only the settings values that you need. You can see how it is done in [this class](https://github.com/mailpoet/mailpoet/blob/master/lib/Config/Menu.php) 
It is not ideal solution, but I would go with that for now.
When you set these values to template in PHP you can also fix the boolean data types and then in JS code you can use `true` and `false`.

Reply:
Look at the last commit, I changed what is passed into template, with possibility to extend it for more polls. I left `true`/`false` values as they were, since I still have to set them both in PHP and JS, so sticking with `"1"` and `"0"`